cmd/cosign/cli/options/certificate.go

```diff
@@ -82,3 +81,13 @@ func (o *CertVerifyOptions) AddFlags(cmd *cobra.Command) {
 		"when set, verification will not check that a certificate contains an embedded SCT, a proof of "+
 			"inclusion in a certificate transparency log")
 }
+
+func (o *CertVerifyOptions) Identities() ([]cosign.Identity, error) {
+	if o.CertIdentity == "" {
+		return nil, errors.New("--certificate-identity is required for verification in keyless mode")
+	}
+	if o.CertOidcIssuer == "" {
+		return nil, errors.New("--certificate-oidc-issuer is required for verification in keyless mode")
+	}
+	return []cosign.Identity{{IssuerRegExp: o.CertOidcIssuer, Issuer: o.CertOidcIssuer, SubjectRegExp: o.CertIdentity, Subject: o.CertIdentity}}, nil
```

Comment:
We don't need to set `IssuerRegExp` and `SubjectRegExp` here, since the case statement just ignores them if they're not set. 

I think we'd need another set of flags to support verifying by regex (for example, an email could contain a period), but I think we could add this later on. Given this is a CLI interface, I don't expect needing to verify a regex.

EDIT: I see you are leveraging these flags as a regex later on. Do we want to just go ahead and add flags for certificate-identity-regex and certificate-oidc-issuer-regex now?

Reply:
Done. Added the new flags.